libsolidity/analysis/DocStringTagParser.cpp

```diff
@@ -211,6 +211,12 @@ bool DocStringTagParser::visit(InlineAssembly const& _assembly)
 								"otherwise only use the NatSpec tag."
 							);
 						_assembly.annotation().markedMemorySafe = true;
+						m_errorReporter.warning(
+							2424_error,
+							_assembly.location(),
+							"Natspec 'memory-safe-assembly' special comment for inline assembly is deprecated and scheduled for removal in the next breaking version (0.9). "
+							"Instead, specify the assembly block annotation."
```

Comment:
```suggestion
							"Use the memory-safe block annotation instead."
```

Also note that the other warning above calls it "assembly flag". This sounds weird to me and is not how we call it in the docs, so I think we should update that comment too.